conf/layer.conf

```diff
@@ -15,3 +15,8 @@ BBFILE_COLLECTIONS += "tracing-layer"
 BBFILE_PATTERN_tracing-layer = "^${LAYERDIR}/"
 BBFILE_PATTERN_IGNORE_EMPTY_tracing-layer = "1"
 BBFILE_PRIORITY_tracing-layer = "1"
+
+LAYERSERIES_COMPAT_tracing-layer = "dunfell"
+
+# BBMASK gstreamer bbappend until its tracing patches get updated
+BBMASK += "${LAYERDIR_tracing-layer}/recipes-multimedia/gstreamer/gstreamer1.0_1.16%.bbappend"
```

Comment:
This will cause masking of appends *outside* this layer, as LAYERDIR_tracing-layer is undefined, meaning the pattern added to BBMASK is actually `/recipes-multimedia/gstreamer/gstreamer1.0_1.16%.bbappend`, which would match against any layer with that path.

Reply:
To clarify, bitbake doesn't implicitly define the LAYERDIR_layername variables, they're explicitly added to layer.conf files, i.e.:
```
LAYERDIR_tracing-layer = "${LAYERDIR}"
```